contents/docs/getting-started/user-properties.mdx

```diff
@@ -53,25 +53,24 @@ posthog.capture(
 
 User property values can be strings, booleans, numbers, objects, or arrays. Note that for objects and arrays, you'll need to use the [Property Flattener app](apps/property-flattener) in order to flatten nested properties and make them accessible through filters in the PostHog app.
```

Comment:
I removed the user properties section from the identify docs. However, since I made some small improvements to the text, I'm including these updates in this PR